<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pandora</class>
 <widget class="QMainWindow" name="Pandora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="gen3FRLGE">
     <attribute name="title">
      <string>Gen III FRLGE</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxSearchBy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Search By</string>
      </property>
      <widget class="QLabel" name="labelPID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PID</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelTID">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>TID</string>
       </property>
      </widget>
      <widget class="QTextBox" name="textBoxPID">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBox" name="textBoxTID">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxParameters">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>231</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Search Parameters</string>
      </property>
      <widget class="QLabel" name="labelMaxFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Max Frame</string>
       </property>
      </widget>
      <widget class="QTextBox" name="textBoxMaxFrame">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="labelMinFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Min Frame</string>
       </property>
      </widget>
      <widget class="QTextBox" name="textBoxMinFrame">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonFindFRLGE">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Find ID Frames</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>231</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTableView" name="tableViewGenerator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>711</width>
      <height>211</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
